catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/metacard/notes/NoteAttributes.java

```diff
@@ -29,14 +30,7 @@
 
   static {
     Set<AttributeDescriptor> descriptors = new HashSet<>();
-    descriptors.add(
-        new AttributeDescriptorImpl(
-            NoteConstants.PARENT_ID,
-            true /* indexed */,
-            true /* stored */,
-            true /* tokenized */,
-            false /* multivalued */,
-            BasicTypes.STRING_TYPE));
+    descriptors.add(new AssociationsAttributes().getAttributeDescriptor(NoteConstants.PARENT_ID));
```

Comment:
❓ Should `NoteConstants.PARENT_ID` be defined in `AssociationsAttributes`? I don't see it.

Reply:
PARENT_ID and Associations.RELATED both map to Metacard.RELATED